skglm/tests/test_group.py

```diff
@@ -132,6 +132,44 @@ def test_vs_celer_grouplasso(n_groups, n_features, shuffle):
     np.testing.assert_allclose(model.coef_, w, atol=1e-5)
 
 
+@pytest.mark.parametrize("n_groups, n_features, shuffle",
+                         [[15, 50, False]])
```

Comment:
if a single set of parameters is used, we can get rid of parametrize and hardcode these.